package/piksi_leds/src/led_adp8866.c

```diff
@@ -318,6 +352,11 @@ bool led_adp8866_leds_set(const led_adp8866_led_state_t *led_states,
   u32 modified_states_count =
       modified_states_get(led_states, led_states_count, modified_states);
 
+  if (flush_led_state_cache) {
+    led_cache_flush();
```

Comment:
Do we care if there's an error here?

Reply:
Possibly, but at this point I'm not exactly sure how we would want to handle that error in a meaningful way. Simply leaving the flush flag set didn't seem like the proper thing to do so I've left it as is for now.